integration_tests/e2e/overview/postInductionOverview.cy.ts

```diff
@@ -76,10 +79,55 @@ context('Prisoner Overview page - Post Induction', () => {
       .isForPrisoner(prisonNumber)
       .isPostInduction()
       .activeTabIs('Overview')
-      .hasGoalsDisplayed()
+      .hasNumberOfGoals(1)
       .hasGoalNotesExpander()
   })
 
+  it('should display prisoner Goals in order of target date, soonest first', () => {
+    // Given
+    cy.task('stubSignInAsUserWithViewAuthority')
+
+    cy.signIn()
+    const prisonNumber = 'G6115VJ'
+
+    const aGoalThatIsDueLater = {
+      ...aValidGoalResponse(),
+      goalReference: uuidv4(),
+      title: 'I am due later',
+      targetCompletionDate: '2025-01-01T00:00:00.000Z',
+    }
+    const aGoalThatIsDueSooner = {
+      ...aValidGoalResponse(),
+      goalReference: uuidv4(),
+      title: 'I am due sooner',
+      targetCompletionDate: '2024-12-25T00:00:00.000Z',
+    }
+    const aGoalThatIsDueSometimeBetween = {
+      ...aValidGoalResponse(),
+      goalReference: uuidv4(),
+      title: 'I am due sometime between the others',
+      targetCompletionDate: '2024-12-29T00:00:00.000Z',
+    }
+    cy.task('getGoalsByStatus', {
+      status: GoalStatusValue.ACTIVE,
+      goals: [aGoalThatIsDueLater, aGoalThatIsDueSooner, aGoalThatIsDueSometimeBetween],
+    })
+
+    // When
+    cy.visit(`/plan/${prisonNumber}/view/overview`)
+
+    // Then
+    const poverviewPage = Page.verifyOnPage(OverviewPage)
+    poverviewPage //
+      .isForPrisoner(prisonNumber)
+      .isPostInduction()
+      .activeTabIs('Overview')
+      .hasNumberOfGoals(3)
+      .goalSummaryCardAtPositionContains(0, aGoalThatIsDueSooner.title)
```

Comment:
Minor, but in similar methods in the Page subclasses, we normally use 1 based indexes (rather than zero based indexes), because it makes reading the tests here a little easier.
IE. here you are referring to the goal at position zero. 
Could we change the `goalSummaryCardAtPositionContains` in Overviewpage.ts` so that it is 1 indexed and is therefore consistent with similar methods?

Reply:
I can do but it uses the cypress accessors now instead of manual numbering. So to make it 1 indexed I'd need to do a `-1` in the page object, which is fine, but feels a bit pointless. LMK if you still want me to change it and I will. 
It's already 0 indexed in the archive goals page so I'd have to update that as well to keep it consistent.

Reply:
I'm not sure its pointless as such; it's about readability of the tests.
But dont worry about it for now, especially as there are others that might also need changing.
Perhaps I'll create a dev tech-debt ticket to address it one day (might be good candidate ticket for any new dev as and when to introduce them to our codebase and the cypress tests 👍 )